soh/soh/Enhancements/presets.h

```diff
@@ -66,221 +66,12 @@ const std::vector<const char*> enhancementsCvars = {
     CVAR_SETTING("DpadInText"),
     CVAR_SETTING("OcarinaControl.Dpad"),
     CVAR_SETTING("OcarinaControl.RStick"),
```

Comment:
We should just get rid of the enhancement and randomizer cvar lists and only clear by blocks. The ones that were in this before shouldn't really have been reset anyway, and if any of these settings becomes a concern for racing down the line, they shouldn't live in settings anyway.

Means you can remove them where Malk placed a comment too.

Reply:
Should I also remove from presets things which don't fall into the blocks? ie vanillaPlusPresets sets `CVAR_SETTING("DpadInText")`, `CVAR_SETTING("OcarinaControl.Dpad")`, `CVAR_SETTING("OcarinaControl.RStick")`

Even randomizer, which only cleared randomizer blocks to begin with, touches settings/enhancements. I saw someone complain about not wanting to touch rando presets because of fear of their non-rando settings getting screwed up. But this might also be fine to punt on for now here since there's the follow up of #4851

Reply:
Yeah I don't think we should in the future. The problem with randomizer presets was it was also touching enhancements though, not just settings. But maybe it's a good rule to follow from now on. Enhancement presets can only touch stuff inside enhancements (and cheats but that'll move to the enhancement tab in the new menu anyway), randomizer preset can only touch randomizer settings etc. 

I do foresee some settings we'd probably want to enable in these presets though, but we can hold a discussion if those need to be moved to enhancements later then.